assemblyline/common/custom.yara

```diff
@@ -589,18 +589,35 @@ rule code_ps1 {
     condition:
         (
             mime startswith "text"
-            and
-                (
-                    2 of ($strong_pwsh*)
-                    or
-                    3 of them
-                )
-        ) or (
+            and (
+                2 of ($strong_pwsh*)
+                or 3 of them
+            )
+        )
+        or (
             mime == "application/octet-stream"
             and 3 of ($strong_pwsh*)
         )
 }
 
+rule code_ps1_first_line {
+
+    meta:
+        type = "code/ps1"
+        score = 5
+
+    strings:
```

Comment:
for the sake of things getting out of sync, can `$strong_pwsh` and `$powershell` both be constants that `code_ps1`, `code_batch` and `code_ps1_first_line` all use?